plugins/spy.yaml

```
@@ -0,0 +1,38 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: spy
+spec:
+  homepage: https://github.com/huazhihao/kubespy
+  shortDescription: pod debugging tool for kubernetes clusters with docker runtimes
```

It's better to stay below ~50chars. Would that work too?
```suggestion
  shortDescription: Debug pods on clusters with docker runtime
```